Gopkg.toml

```diff
@@ -31,7 +31,8 @@
 
 [[constraint]]
   name = "github.com/cloudtrust/keycloak-client"
-  version = "=v1.0-rc3"
+  #version = "=v1.0-rc3"
+  branch="CLOUDTRUST-1617_updateAccount"
```

Comment:
Reminder: to be updated before merging this PR (and remove equal sign in version definition)